core/src/main/java/in/testpress/core/TestpressSessionManager.java

```diff
@@ -0,0 +1,82 @@
+package in.testpress.core;
```

Comment:
Rename this internal `core` folder as `network` and move this file outside and rename file as `TestpressSdk`